presets/aura/accordioncontent/index.js

```diff
@@ -1,3 +1,16 @@
 export default {
-	content: 'p-6 pb-5 pt-0 text-bluegray-600 dark:text-surface-0',
+	content: {
+		class: [
+			'pt-2 text-sm font-normal leading-[18px] text-bluegray-900',
+			'overflow-hidden',
```

Comment:
Not sure about this part, but maybe it'd make more sense as an override in the place where the component is used? We're generally trying to keep the preset modifications as small as possible.